openmls/src/group/tests/external_remove_proposal.rs

```diff
@@ -30,7 +30,10 @@ fn new_test_group(
     let mls_group_config = MlsGroupCreateConfig::builder()
         .wire_format_policy(wire_format_policy)
         .crypto_config(CryptoConfig::with_default_version(ciphersuite))
-        .external_senders(external_senders)
+        .with_group_context_extensions(Extensions::single(Extension::ExternalSenders(
```

Comment:
Nothing to change here. But I noticed that we're not consistent using `with_property` or only `property` in builders.

Reply:
True, but that's a lot of additional changes that I'd rather have in a separate PR. We'll also have to agree on a style. Maybe generally something we should do once we're done with refactoring the builders.